### ISSUE TITLE
[TBC - Shadowmoon Valley - NPC] Warcaller Beersnout

### ISSUE THREAD

**Issue by Cythisiax** [azerothcore/azerothcore-wotlk#20706]

**Comment by TheSCREWEDSoftware** [comment]
CC: https://github.com/chromiecraft/chromiecraft/issues/5583

Warcaller doesn't move, should it? can't find a source saying he should or not. We needs sources and sniff for the data. he does talk as it's shown in the wowhead link above.

- `go c i 21775` to see the npc not moving

**Comment by Cythisiax** [comment]
> CC: [chromiecraft/chromiecraft#5583](https://github.com/chromiecraft/chromiecraft/issues/5583)
> 
> Warcaller doesn't move, should it? can't find a source saying he should or not. We needs sources and sniff for the data. he does talk as it's shown in the wowhead link above.
> 
> * `go c i 21775` to see the npc not moving

Nope! He does not indeed move, he just stands there right in front of the bounty board!
